### PR TITLE
Change background color of platonic upgrades if they are partially or fully upgraded.

### PR DESCRIPTION
**Pull request by eyebrowsoffire**

### CODE REVIEW

File: src/Synergism.ts [hunk 1]
```diff
@@ -1119,6 +1120,9 @@ const loadSynergy = (reset = false) => {
         for (let j = 1; j <= 50; j++) {
             updateCubeUpgradeBG(j);
         }
+        for (let j = 1; j <= 20; j++) {
```
Comment (KhafraDev) [inline src/Synergism.ts:1123]:
```suggestion
        const platUpg = document.querySelectorAll('img[id^="platUpg"]');
        for (let j = 1; j <= platUpg.length; j++) {
```

that way other elements can be added in the future without having to change the hardcoded number of elements

File: src/Platonic.ts [hunk 2]
```diff
@@ -329,6 +329,22 @@ export const createPlatonicDescription = (index: number) => {
...
 }
 
+export const updatePlatonicUpgradeBG = (i: number) => {
+    const a = document.getElementById("platUpg" + i)
+
+    const maxLevel = platUpgradeBaseCosts[i].maxLevel
+    if (player.platonicUpgrades[i] === 0) {
+        a.style.backgroundColor = "black"
+    }
+    if (player.platonicUpgrades[i] > 0 && player.platonicUpgrades[i] < maxLevel) {
```
Comment (KhafraDev) [inline src/Platonic.ts:339]:
```suggestion
    else if (player.platonicUpgrades[i] > 0 && player.platonicUpgrades[i] < maxLevel) {
```

File: src/Platonic.ts [hunk 3]
```diff
@@ -329,6 +329,22 @@ export const createPlatonicDescription = (index: number) => {
...
+        a.style.backgroundColor = "purple"
+    }
+    if (player.platonicUpgrades[i] === maxLevel) {
```
Comment (KhafraDev) [inline src/Platonic.ts:342]:
```suggestion
    else if (player.platonicUpgrades[i] === maxLevel) {
```

File: src/Platonic.ts [hunk 4]
```diff
@@ -329,6 +329,22 @@ export const createPlatonicDescription = (index: number) => {
     }
 }
 
+export const updatePlatonicUpgradeBG = (i: number) => {
+    const a = document.getElementById("platUpg" + i)
```
Comment (KhafraDev) [inline src/Platonic.ts:333]:
```suggestion
    const a = document.getElementById(`platUpg${i}`)
```

**Review by KhafraDev — CHANGES_REQUESTED** [review]

**Comment by eyebrowsoffire** [comment]
Applied your suggestions and restested the changes.

**Review by KhafraDev — APPROVED** [review]

**Comment by Pseudonian** [comment]
Looks solid. Great work :)

**Pull request merged**